source/user/installation_guide.rst

```diff
@@ -112,6 +70,55 @@ The last step is provisioning a directory where data will be stored. Copy paste
 You should now be able to run a scenario, per the following section.
 Note: when doing so, your working directory should be the root of the PowerSimData repository.
 
+Client-Server Installation
+--------------------------
+Here is what needs to be done to use this framework.
+
+
+On Laptop/Desktop
++++++++++++++++++
++ Fork `PowerSimData <https://github.com/Breakthrough-Energy/PowerSimData>`_ (see our
+  :doc:`git_guide` guide to set this up)
++ Generate authentication keys (SSH keys) using the ``ssh-keygen`` program
++ Create a **PowerSimData/powersimdata/utility/.server_user** file enclosing your
+  username on the server if different from your local username
++ Edit **PowerSimData/powersimdata/utility/server_setup.py** file as follows:
+    + Set ``SERVER_ADDRESS``: your server address
+    + Set ``DATA_ROOT_DIR``: root of the file system
+    + Set ``EXECUTE_DIR``: scenario inputs and outputs will be stored there - note that
+      these files are temporary files and are not necessary to load an already ran
+      scenario
+    + Set ``INPUT_DIR``: all scenario inputs are stored there - after the scenario has
+      finished running and the temporary data are extracted, the **MAT-file** that
+      encloses the grid used for the scenario is stored there along with the **pickle**
+      file enclosing the change table listing the transformation applied to the
+      original grid
+    + Set ``OUTPUT_DIR``: all scenario outputs are stored there after being extracted -
+      the list of output data is given in the :doc:`../powersimdata/index` tutorial
+    + Set ``MODEL_DIR``: this directory encloses the simulation engine
+    + Set ``LOCAL_DIR``: this directory will enclose your local file system
+    + Note that ``BACKUP_DATA_ROOT_DIR`` does not need to be set unless you have a
+      backup storage mounted on you server that you plan to use to transfer scenario
+      data.
```

Comment:
This has changed. Perhaps @jon-hagg can help here but the above variables are now partly set in the `powersimdata.utility.config` module

Reply:
Not sure the best way to clarify things for a new user, but the guide is still partly accurate - the variables are defined in the config module but redefined in server_setup based on the environment, so a user could still override them there as described. Maybe it would be helpful to mention something like "if you want to see the default values for reference, they are in `config.py`"?

Reply:
So you would recommend that a new user sets the variables in `setup.py` and does not touch the default values in `config.py`?

Reply:
For now yeah that is probably the simplest way to do it. I think we can also leave the docs as-is, unless we want to add the note about the defaults in `config.py`. Not sure how important it is given I think we are the only ones using the client server installation.